atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/indexing/generated/DataTable.java

```diff
@@ -12,16 +10,11 @@
 import java.util.SortedMap;
 import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutorService;
-import java.util.concurrent.TimeUnit;
```

Comment:
I did it as a bulk action.  It will probably come up again next time code gen is run, but for now I think it is fine?